### PR TITLE
Thrifty Lambda functions

### PR DESCRIPTION
**Pull request by sankeyraut**
### Checklist
- [ ] Issue(s) linked

Added controls for lambda functions.

### CODE REVIEW

File: query/lambda/excessive_timout.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  end as status,
+  case
+    when avg_duration is null then 'CloudWatch metrics not available for ' || title || '.'
+    else title || ' Timeout of ' || timeout::numeric*1000 || ' milliseconds is ' || round(((timeout :: numeric*1000)-avg_duration)/(timeout :: numeric*1000)*100,1) || '% more as compared to average of ' || round(avg_duration,0) || ' milliseconds.'
+  end as reason,
+  region,
+  account_id
+from
+  aws_lambda_function f
+  left join lambda_duration as d on f.name = d.name
```
Comment (rajlearner17) [inline query/lambda/excessive_timout.sql:27]: left join lambda_duration as d on f.name = d.name;
Add a semicolon at the end of the queries

File: query/lambda/excessive_timout.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+)
+select
+  arn as resource,
+  case
+    when avg_duration is null then 'error'
+    when ((timeout :: numeric*1000)-avg_duration)/(timeout :: numeric*1000) > 0.1 then 'alarm'
+    else 'ok'
+  end as status,
+  case
+    when avg_duration is null then 'CloudWatch metrics not available for ' || title || '.'
```
Comment (rajlearner17) [inline query/lambda/excessive_timout.sql:20]: The reason can be confusing if we don't mention the lambda in it 
'CloudWatch metrics not available for ' > CloudWatch Lambda function metrics not available for ...

File: controls/lambda.sp [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+benchmark "lambda" {
+  title         = "Lambda Checks"
+  description   = "Thrifty developers ensure their Lambda functions are optimized."
+  documentation = file("./controls/docs/lambda.md")
+  tags          = local.lambda_common_tags
+  children = [
+    control.high_error_rate,
+    control.excessive_timout,
+  ]
```
Comment (rajlearner17) [inline controls/lambda.sp:15]: can you pls sort this and remove 1 comma

```
control.excessive_timout,
control.high_error_rate
```

File: query/lambda/excessive_timout.sql (hunk 1 above)
Comment (sankeyraut) [reply query/lambda/excessive_timout.sql]: Done.

File: query/lambda/excessive_timout.sql (hunk 2 above)
Comment (sankeyraut) [reply query/lambda/excessive_timout.sql]: Done

File: controls/lambda.sp (hunk 3 above)
Comment (sankeyraut) [reply controls/lambda.sp:15]: Done

**Review by rajlearner17 — APPROVED** [review]
LGTM

**Pull request merged**